### PR TITLE
revert to logstash-core-event 

### PR DESCRIPTION
**Pull request by colinsurprenant**
logstash-core-event-java introduces backward compatibility problems with ruby filter scripts and custom plugins.

before merging, these 2 specs errors need to be addressed

```
  1) LogStash::Timestamp at with BigDecimal epoch should return usec with a minimum of millisec precision
     Failure/Error: expect(LogStash::Timestamp.at(BigDecimal.new("946702800.123456")).usec).to be_within(1000).of(123456)
       expected 0 to be within 1000 of 123456
     # ./logstash-core-event/spec/logstash/timestamp_spec.rb:144:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.8/lib/rspec/wait.rb:46:in `(root)'
     # ./rakelib/test.rake:42:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:248:in `execute'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:243:in `execute'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:187:in `invoke_with_call_chain'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:180:in `invoke_with_call_chain'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:173:in `invoke'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:150:in `invoke_task'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:106:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:106:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:115:in `run_with_threads'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:100:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:78:in `run'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:176:in `standard_exception_handling'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:75:in `run'

  2) LogStash::Timestamp at with BigDecimal epoch should convert to correct date
     Failure/Error: expect(LogStash::Timestamp.at(BigDecimal.new("946702800.123456")).to_iso8601).to eq("2000-01-01T05:00:00.123Z")

       expected: "2000-01-01T05:00:00.123Z"
            got: "2000-01-01T05:00:00.000Z"

       (compared using ==)
     # ./logstash-core-event/spec/logstash/timestamp_spec.rb:138:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.8/lib/rspec/wait.rb:46:in `(root)'
     # ./rakelib/test.rake:42:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:248:in `execute'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:243:in `execute'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:187:in `invoke_with_call_chain'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:180:in `invoke_with_call_chain'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/task.rb:173:in `invoke'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:150:in `invoke_task'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:106:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:106:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:115:in `run_with_threads'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:100:in `top_level'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:78:in `run'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:176:in `standard_exception_handling'
     # ./vendor/bundle/jruby/1.9/gems/rake-11.1.2/lib/rake/application.rb:75:in `run'

Finished in 16.5 seconds (files took 3.21 seconds to load)
1441 examples, 2 failures

Failed examples:

rspec ./logstash-core-event/spec/logstash/timestamp_spec.rb:141 # LogStash::Timestamp at with BigDecimal epoch should return usec with a minimum of millisec precision
rspec ./logstash-core-event/spec/logstash/timestamp_spec.rb:137 # LogStash::Timestamp at with BigDecimal epoch should convert to correct date

Randomized with seed 12654
```

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -10,7 +10,7 @@ PATH
       i18n (= 0.6.9)
```
Comment (talevy) [inline Gemfile.jruby-1.9.lock]: can we revert these local-build changes to the lock file?

Comment (talevy) [inline Gemfile.jruby-1.9.lock]: can we revert these local-build changes to the lock file?

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: no, no reason to not have them up to date

**Comment by colinsurprenant** [comment]
PR #5024 is required to fix the mentioned specs errors

File: Gemfile [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 source "https://rubygems.org"
 gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
-# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
-gem "logstash-core-event-java", "3.0.0.dev", :path => "./logstash-core-event-java"
+gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```
Comment (andrewvc) [inline Gemfile]: @colinsurprenant  Is this the right version? Shouldn't this be a 2.x event?

Comment (colinsurprenant) [inline Gemfile]: well, this is what we have in 2.x, when we backport in 2.3 we have to adjust anyway so, I don't see the need to change that here at this point.

**Comment by ph** [comment]
LGTM

File: Gemfile.jruby-1.9.lock (hunk 1 above)
Comment (ph) [inline Gemfile.jruby-1.9.lock]: @talevy I see no reason for that, anyway we will have to generate a new one for 2.3 and 2.2 releases.

**Comment by colinsurprenant** [comment]
please jenkins, test this

**Comment by colinsurprenant** [comment]
green, merging.

**Comment by colinsurprenant** [comment]
merged in 2.x

**Pull request merged**